### PR TITLE
Tex sources for reco & trigger chapter

### PR DESCRIPTION
**Pull request by particleist**

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-14_recotrigger/a2tex.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
+#! /usr/bin/env python
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-14_recotrigger/a2tex.py]: This is a copied version of what @jouvin has for the main document - is it really needed?

Comment (particleist) [reply CWP/papers/HSF-CWP-2017-14_recotrigger/a2tex.py]: Probably not indeed -- I was following the principle of minimal cognitive effort when preparing this MR.

**Comment by graeme-a-stewart** [comment]
OK, I removed the file from my branch that's the PR to yours (https://github.com/particleist/documents/pull/1). You accept that one, which will update the source branch for this one, then I can accept this one!

**Comment by graeme-a-stewart** [comment]
Great - accepting a squashed version now!

**Pull request merged**